workspaces/nexus-repository-manager/plugins/nexus-repository-manager/catalog-info.yaml

```diff
@@ -20,8 +20,7 @@ metadata:
 spec:
   type: backstage-plugin
   lifecycle: production
-  owner: owner-unknown
-  system: rhdh
+  owner: rhdh-team
```

Comment:
I see that you are updating both the workspace catalog info files and the individual plugin catalog-info files. Do you think that this should be split and an update make to the [nexus repo manager workspace catalog info file](https://github.com/backstage/community-plugins/blob/main/workspaces/nexus-repository-manager/catalog-info.yaml) to make it more consistent?

Reply:
Yes, I think I should update keycloak and nexus as well.

Reply:
None if these exist:

```
find . -iname catalog-info.yaml | xargs grep -h 'sonarqube.org/project-key' | sort | uniq -c | sort -nr
     22     sonarqube.org/project-key: backstage-community_plugins
      5     sonarqube.org/project-key: backstage-community_backstage-plugins
      2     sonarqube.org/project-key: backstage_community-plugins
      2     sonarqube.org/project-key: backstage/community-plugins
```

I will remove all these annotations.

Reply:
Done.